stdlib/2and3/threading.pyi

```diff
@@ -40,10 +40,10 @@ if sys.version_info >= (3,):
 class ThreadError(Exception): ...
 
 
-# TODO: Change to a class with __getattr__ and __setattr__
-# once mypy supports universal __setattr__.
-# See https://github.com/python/mypy/issues/521
-local = ...  # type: Any
+class local:
```

Comment:
We should make this inherit from `object` in case any type checkers care about the difference between old-style and new-style classes in Python 2. (I checked and `issubclass(threading.local, object)` returns true in 2.7.)

Reply:
Ah, thanks. Fixed.